src/drivers/driver_commands.cpp

```diff
@@ -30,6 +32,15 @@ etl::optional<int> handle_command(DriverCommand type, int arg1, int arg2, int ar
                 static_cast<uint32_t>(arg1)
             );
             break;
+        case DriverCommand::SET_PIN:
+            edge::aidan::set_gpio_pin(static_cast<uint32_t>(arg1));
```

Comment:
I'd rather have an actual driver for GPIO vs driver_commands directly interacting with the HAL

Reply:
i think this is fine for the illustrative purpose